airbyte_cdk/sources/declarative/parsers/model_to_component_factory.py

```diff
@@ -2618,6 +2613,8 @@ def create_gzip_decoder(
             fallback_parser=gzip_parser.inner_parser,
         )
 
+    # todo: This method should be removed once we deprecate the SimpleRetriever.cursor field and the various
```

Comment:
nit: I feel this comment would be more actionable if it were in the SimpleRetriever